python_modules/dagster/dagster/_core/execution/job_backfill.py

```diff
@@ -88,6 +104,7 @@ def execute_job_backfill_iteration(
                 # before submitting, refetch the backfill job to check for status changes
                 backfill = cast(PartitionBackfill, instance.get_backfill(backfill.backfill_id))
                 if backfill.status != BulkActionStatus.REQUESTED:
+                    tick_context.update_state(TickStatus.SUCCESS)
```

Comment:
since the backfill has been marked as CANCELED or CANCELING and we are gracefully handling that in the daemon, i think that means the tick is successful